### PR TITLE
Add foreman-docker 3.x version manual

### PR DESCRIPTION
**Pull request by bastilian**
including compatibility information

### CODE REVIEW

File: plugins/foreman_docker/3.x/index.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+You can use the Foreman installer to install this plugin. Use the following command:
+    foreman-installer --enable-foreman-plugin-docker
+
+If you prefer not to use the installer, follow the instructions below for your operating system.
+
+## 3.2 Red Hat, CentOS, Fedora, Scientific Linux (rpm)
+
+Set up the repo as explained in the link above, then run
+
+    # yum install ruby193-rubygem-foreman_docker
```
Comment (mmoll) [inline plugins/foreman_docker/3.x/index.md]: that's old :)

File: plugins/foreman_docker/3.x/index.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+    <tr>
+      <td align="right">&gt;= 3.1.0</td>
+      <td align="right"><a href="http://docs.master.dockerproject.org/v1.7/reference/api/registry_api/">v1</a>, <a href="https://docs.docker.com/registry/spec/api/">v2</a><strong>*</strong></td>
+    </tr>
+  </tbody>
+</table>
+
+**Note:* API v2 as default and v1 to fall back on.
+
+
```
Comment (mmoll) [inline plugins/foreman_docker/3.x/index.md]: superfluous newline

File: plugins/foreman_docker/3.x/index.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+      <td align="right"><a href="http://docs.master.dockerproject.org/v1.7/reference/api/registry_api/">v1</a></td>
+    </tr>
+    <tr>
+      <td align="right">&gt;= 3.1.0</td>
+      <td align="right"><a href="http://docs.master.dockerproject.org/v1.7/reference/api/registry_api/">v1</a>, <a href="https://docs.docker.com/registry/spec/api/">v2</a><strong>*</strong></td>
+    </tr>
+  </tbody>
+</table>
+
+**Note:* API v2 as default and v1 to fall back on.
```
Comment (mmoll) [inline plugins/foreman_docker/3.x/index.md]: API v2 is default, v1 is used as fallback

File: plugins/foreman_docker/3.x/index.md [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+
+## 3.1 Foreman Installer (recommended)
+You can use the Foreman installer to install this plugin. Use the following command:
+    foreman-installer --enable-foreman-plugin-docker
+
+If you prefer not to use the installer, follow the instructions below for your operating system.
+
+## 3.2 Red Hat, CentOS, Fedora, Scientific Linux (rpm)
+
+Set up the repo as explained in the link above, then run
```
Comment (mmoll) [inline plugins/foreman_docker/3.x/index.md]: which link? 😖

File: plugins/foreman_docker/3.x/index.md [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+## 4.1 Managing containers
+
+The following sections show how to create, view, start, stop, and commit a container.
+
+### ⁠Prerequisites
+
+In Foreman, you can deploy containers only on a compute resource of the Docker provider type. Therefore, when you attempt to view or create containers for the first time, Foreman prompts you to create a Docker compute resource. If you don't have a Docker host available, first create a container host as described in Procedure 1.1, “To Prepare a Container Host:”, then specify this host as a compute resource as described in Procedure 1.2, “To Create a Docker Compute Resource:”.
+
+
+**Procedure 1.1. To Prepare a Docker Host:**
```
Comment (mmoll) [inline plugins/foreman_docker/3.x/index.md]: Is there the possibility to use `####` as headlining here with the current github-pages markdown? But I'm unsure if that's more or less confusing than the current "inline" numbering.

**Review by mmoll — CHANGES_REQUESTED** [review]

File: plugins/foreman_docker/3.x/index.md [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+**Procedure 1.7. To remove a container:**
+
+1. Navigate to Containers → All Containers to view the list of available containers.
+2. Click the name of the container you want to delete.
+3. Click Delete.
+4. In the alert box, click OK to remove the container.
+⁠
+
+## 4.3. Working with repositories
+Follow the Katello [Docker documentation](http://www.katello.org/docs/user_guide/docker/docker.html).
```
Comment (mmoll) [inline plugins/foreman_docker/3.x/index.md]: 404

**Comment by bastilian** [comment]
@mmoll Thanks for reviewing! I've updated it and addressed the issues and structured the doc a bit better by making the procedures into headlines.

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, danke @bastilian!

**Pull request merged**